cypress/support/fragments/settings/dataImport/dataImportSettingsMatchProfiles.js

```diff
@@ -0,0 +1,78 @@
+import getRandomPostfix from '../../../utils/stringTools';
```

Comment:
It seems to me that the files dataImportSettingsMatchProfiles.js,dataImportSettingsMappingProfiles.js,dataImportSettingsJobProfiles.js,dataImportSettingsActionProfiles.js can be named without the prefix dataImport.